concept/type/impl/TypeImpl.java

```diff
@@ -92,78 +145,83 @@ public final String getLabel() {
 
         @Override
         public final void setLabel(String label) {
-            final ConceptProto.TypeMethod.Req method = ConceptProto.TypeMethod.Req.newBuilder()
+            final TypeMethod.Req method = TypeMethod.Req.newBuilder()
                     .setTypeSetLabelReq(ConceptProto.Type.SetLabel.Req.newBuilder()
                                                 .setLabel(label)).build();
-            runMethod(method);
+            execute(method);
         }
 
         @Override
         public final boolean isAbstract() {
-            final ConceptProto.TypeMethod.Req method = ConceptProto.TypeMethod.Req.newBuilder()
+            final TypeMethod.Req method = TypeMethod.Req.newBuilder()
                     .setTypeIsAbstractReq(ConceptProto.Type.IsAbstract.Req.getDefaultInstance()).build();
 
-            return runMethod(method).getTypeIsAbstractRes().getAbstract();
+            return execute(method).getTypeIsAbstractRes().getAbstract();
+        }
+
+        void setSupertypeExecute(Type type) {
```

Comment:
See comments on `getSupertype` below - we should align the names of these two methods; either `getSupertypeExecute` and `setSupertypeExecute`, or `getSupertype` and `setSupertype`.

Reply:
Replied above.